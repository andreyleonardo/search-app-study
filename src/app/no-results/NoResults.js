import React from 'react';
import { FormattedMessage } from 'react-intl';

const NoResults = () => (
  <section className="app__no_results col-xs-12">
    <p><FormattedMessage id="no_results.message" /></p>
  </section>
);

export default NoResults;
