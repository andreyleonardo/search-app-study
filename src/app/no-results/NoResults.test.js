import React from 'react';
import NoResults from './NoResults';

describe('No Results', () => {
  it('renders without crash', () => {
    const noResults = shallow(<NoResults />);

    expect(noResults).toMatchSnapshot();
  });
});
