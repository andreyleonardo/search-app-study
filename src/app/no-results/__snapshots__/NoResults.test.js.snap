// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`No Results renders without crash 1`] = `
ShallowWrapper {
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <NoResults />,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateError": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "host",
    "props": Object {
      "children": <p>
        <FormattedMessage
          id="no_results.message"
          values={Object {}}
        />
      </p>,
      "className": "app__no_results col-xs-12",
    },
    "ref": null,
    "rendered": Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": <FormattedMessage
          id="no_results.message"
          values={Object {}}
        />,
      },
      "ref": null,
      "rendered": Object {
        "instance": null,
        "key": undefined,
        "nodeType": "class",
        "props": Object {
          "id": "no_results.message",
          "values": Object {},
        },
        "ref": null,
        "rendered": null,
        "type": [Function],
      },
      "type": "p",
    },
    "type": "section",
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": <p>
          <FormattedMessage
            id="no_results.message"
            values={Object {}}
          />
        </p>,
        "className": "app__no_results col-xs-12",
      },
      "ref": null,
      "rendered": Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": <FormattedMessage
            id="no_results.message"
            values={Object {}}
          />,
        },
        "ref": null,
        "rendered": Object {
          "instance": null,
          "key": undefined,
          "nodeType": "class",
          "props": Object {
            "id": "no_results.message",
            "values": Object {},
          },
          "ref": null,
          "rendered": null,
          "type": [Function],
        },
        "type": "p",
      },
      "type": "section",
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
        "lifecycles": Object {
          "componentDidUpdate": Object {
            "onSetState": true,
          },
          "getDerivedStateFromProps": true,
          "getSnapshotBeforeUpdate": true,
          "setState": Object {
            "skipsComponentDidUpdateOnNullish": true,
          },
        },
      },
    },
  },
}
`;
